Makefile

```diff
@@ -68,6 +68,10 @@ benchmark-all:
 benchmark-pydantic:
 	python benchmarks/run.py pydantic-only
 
+.PHONY: benchmark-json
```

Comment:
the problem with having these addition as part of this PR, is that we can't easily compare the performance with master before merging this PR.

Sorry to be so annoying, but would it be possible to move the json benchmarks to a new PR?

Reply:
👍 Sure thing.

Reply:
Created https://github.com/samuelcolvin/pydantic/pull/1344 for just the benchmarks.